src/index.js

```diff
@@ -1,11 +1,20 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import Home from './components/Home';
+import Quote from './components/Quote';
+import Calculator from './components/Calculator';
+
+const rootElement = document.getElementById('root');
 
 ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-  document.getElementById('root'),
+  <BrowserRouter>
+    <Routes>
+      <Route path="/" element={<Home />} />
+      <Route path="calculator" element={<Calculator />} />
+      <Route path="quotes" element={<Quote />} />
+    </Routes>
+  </BrowserRouter>,
+  rootElement,
```

Comment:
- [ ] Let us try to keep our code clean. Since you are not using the **`App.js`** component at all it is redundant. Kindly delete it.